content/news/this-week-in-fluvio-0008.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "This Week in Fluvio &#x23;8"
+date: 2021-09-30
+weight: 20
+---
+
+Welcome to This Week in Fluvio, our weekly newsletter
+for development updates to [Fluvio open source]. Fluvio is a distributed,
+programmable streaming platform written in Rust.
+
+## Release of Fluvio `0.9.9`
+
+### Fluvio Consumer can now stream from all partitions in a topic
+
+As of this release, you can now use the Fluvio Consumer CLI and Rust API to stream
+records from _all_ partitions in a topic, rather than just one at a time. To use
+this functionality on the CLI, use the new `--all-partitions` flag.
+
+To see this in action, let's produce some records to a topic with 2 partitions.
+First, let's create the topic:
+
+%copy first-line%
+```bash
+$ fluvio topic create fruits -p2
```

Comment:
shouldn't there be a space between `-p` and 2? I noticed it works, but it just looks odd to me.

Reply:
I can add a space there 👍